azurerm/resource_arm_container_group.go

```diff
@@ -108,25 +108,16 @@ func resourceArmContainerGroup() *schema.Resource {
 							}, true),
 						},
 
-						"env_var": {
-							Type:     schema.TypeList,
+						"env_vars": {
+							Type:     schema.TypeMap,
+							Optional: true,
```

Comment:
be aware, if we'd done a release of this, we'd have to include a State Migration to handle converting between the two fields going from a TypeList -> TypeMap - but I think we can probably overlook that this time since this hasn't been released yet

Reply:
👍